README.md

```diff
@@ -72,42 +96,270 @@ Resource actions are indicated with the following symbols:
 
 Terraform will perform the following actions:
 
-  + zendesk_attachment.logo
-      id:                                   <computed>
-      content_type:                         <computed>
-      content_url:                          <computed>
-      file_hash:                            "Vtptw0XCL7+ShQ8G3/9Q2dsYu3iofOk7J3WqTwzning="
-      file_name:                            "street.jpg"
-      file_path:                            "../zendesk/testdata/street.jpg"
-      inline:                               <computed>
-      size:                                 <computed>
-      thumbnails.#:                         <computed>
-      token:                                <computed>
-
-  ...
-
-  + zendesk_trigger.auto-reply-trigger
-      id:                                   <computed>
-      action.#:                             "1"
-      action.14699407.field:                "notification_user"
-      action.14699407.value:                "[\n  \"requester_id\",\n  \"Dear my customer\",\n  \"Hi. This message was configured by terraform-provider-zendesk.\"\n]\n"
-      active:                               "true"
-      all.#:                                "3"
-      all.1027606754.field:                 "update_type"
-      all.1027606754.operator:              "is"
-      all.1027606754.value:                 "Create"
-      all.2406064215.field:                 "role"
-      all.2406064215.operator:              "is"
-      all.2406064215.value:                 "end_user"
-      all.375493961.field:                  "status"
-      all.375493961.operator:               "is_not"
-      all.375493961.value:                  "solved"
-      title:                                "Auto Reply Trigger"
-
-
-Plan: 16 to add, 0 to change, 0 to destroy.
+  # zendesk_attachment.logo will be created
```

Comment:
honestly i think we can do without some of the resources in the readme and just use the ellipses.

Reply:
fixed at 55b83a4